packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -48,6 +48,11 @@ abstract class Renderer extends Object with Scheduler, Services
     handleMetricsChanged(); // configures renderView's metrics
   }
 
+  /// The render tree's owner, which maintains dirty state for layout,
+  /// composite, paint, and accessibility semantics
+  PipelineOwner get pipelineOwner => _pipelineOwner;
+  PipelineOwner _pipelineOwner = new PipelineOwner();
```

Comment:
`final PipelineOwner pipelineOwner = new PipelineOwner();`

Using `final` has three benefits:

* We prevent writes to the _pipelineOwner, even from inside this class.
* The initialization will be delayed until first use instead of eagerly done at construction.  This doesn't matter much for this class because we need to create the PipelineOwner before first paint anyway, but it's a good habit.
* We can skip the extra getter because we having a final public field is the same as just having a public getter.